### ISSUE TITLE
Fix S6964 FP: Property with a default value

### ISSUE THREAD

**Issue by DerGary** [SonarSource/sonar-dotnet#9331]
### Description

[S6964](https://sonarsource.github.io/rspec/#/rspec/S6964/csharp) is reporting an error on properties with a default value

### Repro steps

```
public record Data
{
    public bool SomeProperty { get; init; } = false;
}

```

### Expected behavior

The rule is not reported as the property contains a default value

### Actual behavior

The rule is triggered and a warning is reported

### Known workarounds

ignore the rule

### Related information

* MSBuild / dotnet version
dotnet 8
* SonarScanner for .NET version (if used)
6.2.0
* Operating System
linux

**Comment by zsolt-kolbay-sonarsource** [comment]
Thank you for reporting this. Confirmed as False Positive.
